Admin/Extension/PublishTimePeriodExtension.php

```diff
@@ -45,15 +51,27 @@ public function configureFormFields(FormMapper $formMapper)
             'required' => false,
         );
 
-        $formMapper->with($this->formGroup, array(
-            'translation_domain' => 'CmfCoreBundle',
-            ))
-            ->add('publish_start_date', 'date', $dateOptions, array(
-                'help' => 'form.help_publish_start_date',
-            ))
-            ->add('publish_end_date', 'date', $dateOptions, array(
-                'help' => 'form.help_publish_end_date',
+        if ($formMapper->hasOpenTab()) {
```

Comment:
yes but as you point out .. the method was only added in 2.4, meaning this code would explode on SonataAdminBundle < 2.4 .. my intention is that we can support also 2.3 with the next minor release of CoreBundle